CommonData/column.cpp

```diff
@@ -1236,15 +1236,29 @@ bool Column::setStringValueToRow(size_t row, const std::string & userEntered)
 	
 	double	newDoubleToSet	= EmptyValues::missingValueDouble,
 			oldDouble		= _dbls[row];	
-	bool	itsADouble		= ColumnUtils::getDoubleValue(userEntered, newDoubleToSet);
+	bool	itsADouble		= ColumnUtils::getDoubleValue(userEntered, newDoubleToSet),
+			nothingThereYet	=	_ints.size() == std::count_if(_ints.begin(), _ints.end(), [&](int i)	{ return i == Label::DOUBLE_LABEL_VALUE || i == EmptyValues::missingValueInteger || labelByIntsId(i)->isEmptyValue(); }) 
```

Comment:
Nice but not really efficient: it has to go through the whole column, but in most of the cases already with the first values, it can see that there are already values here.

Reply:
fair enough, let me see